apps/app/pages/admin/event/[id].tsx

```diff
@@ -57,27 +61,65 @@ function EventPage() {
-        <Title level={2}>Mentores disponíveis</Title>
-        <List
-          itemLayout="vertical"
-          dataSource={mentors}
-          renderItem={(mentor: any) => (
-            <List.Item>
-              <Link href={`/profile/mentor/${mentor.id}`}>
-                <List.Item.Meta
-                  avatar={<Avatar size={64} src={mentor.photo} />}
-                  title={`${mentor.first_name} ${mentor.last_name}`}
-                  description={mentor.notes ? `Notas: ${mentor.notes}` : ""}
-                />
-              </Link>
-              <div style={{ fontWeight: "500" }}>
-                {mentor.is_available ? "Disponível" : "Não disponível"}
-              </div>
-            </List.Item>
-          )}
-        />
-      </Col>
+      <>
+        {available ? (
+          <>
+            <Row style={{ display: "flex", justifyContent: "space-between" }}>
+              <Title level={2}>Mentores disponíveis</Title>
+              <Button
+                type="primary"
+                style={{ float: "right" }}
+                onClick={(_) => setAvailable(!available)}
+              >
+                Mentores indisponíveis
+              </Button>
+            </Row>
+            <List
+              itemLayout="vertical"
+              dataSource={availableMentors}
+              renderItem={(mentor: any) => (
+                <List.Item style={{ cursor: "pointer" }}>
+                  <Link href={`/profile/mentor/${mentor.id}`}>
+                    <List.Item.Meta
+                      avatar={<Avatar size={64} src={mentor.photo} />}
+                      title={`${mentor.first_name} ${mentor.last_name}`}
+                      description={mentor.notes ? `Notas: ${mentor.notes}` : ""}
+                    />
+                  </Link>
+                </List.Item>
+              )}
+            />
+          </>
+        ) : (
+          <>
+            <Row style={{ display: "flex", justifyContent: "space-between" }}>
+              <Title level={2}>Mentores indisponíveis</Title>
+              <Button
+                type="primary"
+                style={{ float: "right" }}
+                onClick={(_) => setAvailable(!available)}
+              >
+                Mentores disponíveis
+              </Button>
+            </Row>
+            <List
+              itemLayout="vertical"
+              dataSource={unavailableMentors}
+              renderItem={(mentor: any) => (
+                <List.Item style={{ cursor: "pointer" }}>
+                  <Link href={`/profile/mentor/${mentor.id}`}>
+                    <List.Item.Meta
+                      avatar={<Avatar size={64} src={mentor.photo} />}
+                      title={`${mentor.first_name} ${mentor.last_name}`}
+                      description={mentor.notes ? `Notas: ${mentor.notes}` : ""}
+                    />
+                  </Link>
+                </List.Item>
+              )}
+            />
+          </>
+        )}
+      </>
```

Comment:
I'm sure you can use some inline conditions to not make this code this duplicated 😅

Reply:
It should be extracted to a component :bulb: